server/appengine/src/main/java/who/AppStoreServlet.java

```diff
@@ -0,0 +1,23 @@
+package who;
+
+import java.io.IOException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * Redirects to appropriate app store.
+ *
+ * @author Bob Lee
+ */
+public class AppStoreServlet extends HttpServlet {
+
+  @Override protected void doGet(HttpServletRequest request, HttpServletResponse response)
+      throws IOException {
+    if (request.getHeader("User-Agent").toLowerCase().contains("android")) {
+      response.sendRedirect("https://play.google.com/store/apps/details?id=org.who.WHOMyHealth");
+    } else {
+      response.sendRedirect("https://apps.apple.com/app/id1503458183");
```

Comment:
So search engines will index App Store?    If someone opens this on their laptop they'll also go to apple.   We can go to a marketing page later if neither iOS nor Android.

Reply:
Yep, App Store link is fine in these cases.